master/buildbot/test/unit/test_www_hooks_poller.py

```diff
@@ -60,7 +59,7 @@ def setUpRequest(self, args, options=True, activate=True):
         anotherchangesrc.setServiceParent(master.change_svc)
 
         yield self.request.test_render(self.changeHook)
-        yield util.asyncSleep(0)
+        yield util.asyncSleep(0.1)
```

Comment:
I think the idea here was to let the poller run, even though the http request has already finished.  A sleep certainly does suggest flakiness, though.